go.mod

```diff
@@ -0,0 +1,3 @@
+module example.com/hello
+
+go 1.17
```

Comment:
Was this meant to be in the recipe or did we want this in staged-recipes for some reason?

Reply:
Ah, this was an accidental commit. This will get autogenerated during the tests. I'll revert it on the feedstock.

Reply:
All good 🙂 Submitted PR ( https://github.com/conda-forge/staged-recipes/pull/15978 ) to drop this from staged-recipes